beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSource.java

```diff
@@ -36,19 +37,21 @@ public class ThrottlingSyncSource implements SyncSource {
   private final SyncSource delegate;
 
   private final RateTracker blocksRateTracker;
-  private final RateTracker blobSidecarsRateTracker;
+  private final Optional<RateTracker> blobSidecarsRateTracker;
```

Comment:
In this case I'd like to follow the `NOOP` pattern. Proliferation of Optionals is a form of spaghetti